src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+}
+
+.header__container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo-link {
+  display: block;
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 0;
   margin: 0;
+  display: block;
+  font-weight: 500;
+  height: 60px;
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You've specified 'Roboto' as the font family, but it's a good idea to specify a generic font family as a fallback, in case 'Roboto' fails to load. You can do this by adding a comma after 'Roboto' and then the generic font family name. For example, you could use 'sans-serif' as a fallback.